plugins/com.google.cloud.tools.eclipse.usagetracker/src/com/google/cloud/tools/eclipse/usagetracker/AnalyticsEvents.java

```diff
@@ -55,4 +56,7 @@
 
   public static final String NATIVE_PROJECT = "native.project";
   public static final String MAVEN_PROJECT = "maven.project";
+
+  public static final String MANAGED_CLOUD_SDK = "managed";
+  public static final String USER_CLOUD_SDK = "user.managed";
```

Comment:
It would make more sense if these followed the `CloudSdkManagementOption` `MANUAL` vs `AUTOMATIC`.